modules/ROOT/pages/design-async-api.adoc

```diff
@@ -236,8 +236,90 @@ components:
           email:
             type: string
 ----
+== Example AsyncAPI 2.6.0 Specification (Inline Avro)
 
-== Example AsyncAPI 2.0 specification
+If you created an AsyncAPI specification project, you can replace the initial API specification with following example code to test inlining the API specification fragments:
+
+
+[source,yaml]
+----
+asyncapi: 2.6.0
+info:
+  title: Example AsyncAPI Spec (Inline Avro)
+  version: '1.0.0'
+channels:
+  myChannel:
+    publish:
+      message:
+        schemaFormat: 'application/vnd.apache.avro;version=1.9.0'
+        payload:
+          name: AmericanFlightDataType
+          type: record
+          fields:
+            - name: ID
+              type: int
+              example: 1
+            - name: code
+              type: string
+              example: ER38sd
+            - name: price
+              type: int
+              example: 400
+            - name: departureDate
+              type: string
+              example: 2017/07/26
+            - name: origin
+              type: string
+              example: CLE
+            - name: destination
+              type: string
+              example: SFO
+            - name: emptySeats
+              type: int
+              example: 0
+            - name: plane
+              type:
+                name: Plane
+                type: record
+                fields:
+                  - name: type
+                    type: string
+                  - name: totalSeats
+                    type: int
+----
+== Example AsyncAPI Specification Fragment (Avro Schema)
+
+If you created an Avro Schema fragment, you can replace the initial API fragment with following example code:
+
+[source,yaml]
+----
```

Comment:
We may probably add an example on how to reference a fragment avro from an Async API spec. For example for this case:
```
asyncapi: 2.6.0
info:
  title: Example AsyncAPI Spec (Inline Avro)
  version: '1.0.0'
channels:
  myChannel:
    publish:
      message:
        schemaFormat: application/vnd.apache.avro;version=1.9.0
        payload:
          $ref: AmericanFlightDataType.avsc
```

Reply:
not sure if this is the right place, but if we mention the file extension `.avsc` we could provide a bit more info, considering it is used by convention and not definition (we internally identify an avro fragment by this extension OR by the exchange.json classifier, but the avro spec makes no mention on how to identify an avro file)